opslevel/resource_opslevel_team_tag.go

```diff
@@ -162,12 +162,20 @@ func (teamTagResource *TeamTagResource) Read(ctx context.Context, req resource.R
 		teamIdentifier = data.TeamAlias.ValueString()
 		team, err = teamTagResource.client.GetTeamWithAlias(teamIdentifier)
 	}
-	if err != nil || team == nil {
+	if err != nil {
+		if (team == nil || team.Id == "") && opslevel.IsOpsLevelApiError(err) {
+			resp.State.RemoveResource(ctx)
+			return
+		}
 		resp.Diagnostics.AddError("opslevel client error", fmt.Sprintf("unable to read team (%s), got error: %s", teamIdentifier, err))
 		return
 	}
 	_, err = team.GetTags(teamTagResource.client, nil)
-	if err != nil || team.Tags == nil {
+	if err != nil {
+		if team.Tags == nil && opslevel.IsOpsLevelApiError(err) {
+			resp.State.RemoveResource(ctx)
+			return
+		}
```

Comment:
I think this is another edgecase.  I think we want to error if the team is not found but RemoveResource if the tag is not found